WordPress/src/main/res/values/strings.xml

```diff
@@ -1148,7 +1148,8 @@
     <string name="feedback_form_attachment_already_added">Attachment already added</string>
     <string name="feedback_form_unable_to_create_tempfile">Unable to create temporary file</string>
     <string name="feedback_form_add_attachments">Add attachments</string>
-    <string name="feedback_form_note">If you need support, please get in touch using the \"Contact Support\" screen</string>
+    <string name="feedback_form_note_text">Please note this is not a support form, and we will not be able to reply. If you need support, please use our "</string>
```

Comment:
What is the trailing `"` for?

Reply:
![Screenshot 2024-11-25 at 5 39 05 PM](https://github.com/user-attachments/assets/3f365468-b8eb-48d3-8258-16a924755d79)